pyproject.toml

```diff
@@ -69,6 +70,7 @@ termcolor = "2.0.1"
 [tool.poetry.extras]
 text = ["datasets", "tokenizers", "transformers"]
 vision = ["datasets", "torchvision", "opencv-python"]
+audio = ["datasets", "pretty_midi", "transformers"]
```

Comment:
Please update installation instructions in README.md to contain this additional extra.